### ISSUE TITLE
Windows seems not to skip loaders properly

### ISSUE THREAD

**Issue by piglovesyou** [ardatan/graphql-toolkit#399]
Hi. I'm creating a lib using graphql-toolkit and I found that Windows seems not to skip loaders properly.

Versions:
```
    "@graphql-toolkit/core": "^0.9.0",
    "@graphql-toolkit/graphql-file-loader": "^0.9.0",
    "@graphql-toolkit/json-file-loader": "^0.9.0",
    "@graphql-toolkit/url-loader": "^0.9.0",
```

## Expected

[This code should work](https://github.com/piglovesyou/graphql-let/blob/5bf66098b824af06134f388d9fdefa0c460d64af/src/lib/create-codegen-opts.ts#L32-L34).

[In Linux, it passes perfectly](https://travis-ci.org/piglovesyou/graphql-let/builds/633513851).

## Actual

[It fails only in Windows](https://travis-ci.org/piglovesyou/graphql-let/jobs/633513855#L132-L140).

Related to #167

**Comment by ardatan** [comment]
Could you try with this canary version?
`0.9.1-alpha-39a7101.1+39a7101`

**Comment by piglovesyou** [comment]
@ardatan [I tried it](https://github.com/piglovesyou/graphql-let/blob/use-canary-toolkit/package.json#L21-L24) and [it works!](https://github.com/piglovesyou/graphql-let/runs/376739200#step:4:35) Feel free to close this issue.

**Comment by ardatan** [comment]
You're welcome! 🙂 Let's keep this open until we release a new version.

**Comment by ardatan** [comment]
Available in 0.9.1 :)
